<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.springframework.cache.interceptor.CacheSyncFailureTests" tests="5" skipped="0" failures="0" errors="0" timestamp="2021-01-22T01:37:41" hostname="SUPERMAN" time="0.124">
  <properties/>
  <testcase name="syncWithTwoGetOperations()" classname="org.springframework.cache.interceptor.CacheSyncFailureTests" time="0.036"/>
  <testcase name="unlessSync()" classname="org.springframework.cache.interceptor.CacheSyncFailureTests" time="0.023"/>
  <testcase name="severalCachesWithResolvedSync()" classname="org.springframework.cache.interceptor.CacheSyncFailureTests" time="0.023"/>
  <testcase name="severalCachesSync()" classname="org.springframework.cache.interceptor.CacheSyncFailureTests" time="0.017"/>
  <testcase name="syncWithAnotherOperation()" classname="org.springframework.cache.interceptor.CacheSyncFailureTests" time="0.023"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
